incubator/kafqa/.helmignore

```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+.DS_Store
+# Common VCS dirs
+.git/
+.gitignore
+.bzr/
+.bzrignore
+.hg/
+.hgignore
+.svn/
+# Common backup files
+*.swp
+*.bak
```

Comment:
don't add all files, and whatever needed eg: swp, idea add it to the root folder `.gitignore`

Reply:
These all autogenerated values. I can add the `.idea` in `.gitignore`